templates/taxi/driver_list.html

```diff
@@ -2,6 +2,9 @@
 
 {% block content %}
     <h1>Driver List
+       <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
```

Comment:
According to the checklist, you should use `-` to split words in URL identification parameter `name`, not the `_`. So it should be `driver-create` instead of `driver_create`.